rllib/core/learner/learner.py

```diff
@@ -1300,22 +1318,21 @@ def _update_from_batch_or_episodes(
         if minibatch_size:
             if self._learner_connector is not None:
                 batch_iter = partial(
-                    MiniBatchCyclicIterator,
-                    uses_new_env_runners=True,
-                    num_total_mini_batches=num_total_mini_batches,
+                    MiniBatchCyclicIterator, _uses_new_env_runners=True
```

Comment:
Do we need this here still? I thought we deprecate the hybrid stack?

Reply:
Officially, not yet. PR is still pending ...